includes/apple-exporter/builders/class-components.php

```diff
@@ -30,7 +30,67 @@ protected function build() {
 		// such as if a thumbnail was used from the body.
 		$components = array_merge( $this->meta_components(), $components );
 
-		return $this->group_body_components( $components );
+		$grouped_components = $this->group_body_components( $components );
+		$grouped_components = $this->componentize( $grouped_components );
+
+		return $grouped_components;
+	}
+
+	/**
+	 * Give an array of components in array format, transform
+	 * all that are needed into components
+	 *
+	 * @param array $components
+	 * @return array
+	 * @access private
+	 */
+	private function componentize( $components ) {
+		$heading_items = array();
+		$header_pos = 0;
+
+		foreach ( $components as $i => $component) {
+			if ( 'title' === $component['role'] || 'byline' === $component['role'] ) {
```

Comment:
Looking at this and lines 85-91 below, I don't think it needs to exist here. `cover`, `title` and `byline` are all meta components which automatically get prepended at the top. See line 31 above which adds `$this->meta_components()` before all others. I think any changes required for the new template would go here, if needed.